src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -250,6 +254,26 @@ namespace System.Diagnostics.Tracing
     /// </remarks>
     public partial class EventSource : IDisposable
     {
+
+#if FEATURE_EVENTSOURCE_XPLAT
+        private static readonly EventListener persistent_Listener = InitializePersistentListener();
```

Comment:
I think it would be good to make the naming here a bit more clear.  persistent_Listener doesn't really tell us what this does.

Perhaps calling it the XPlat listener to match the naming scheme you're using elsewhere throughout this PR?